### PR TITLE
BUGFIX: TMA-253 Make sure type of all values in params hash is String

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/217/ : SUCCESS in 24s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/152/ : SUCCESS in 41m 57s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/243/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/178/ : FAILURE in 46m 18s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/244/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/179/ : SUCCESS in 43m 33s

File: lib/gooddata/models/schedule.rb [hunk 1]
```diff
@@ -527,5 +527,24 @@ def to_update_payload
 
       res
     end
+
+    private
+
+    def stringify_values(hash)
```
Comment (korczis) [inline lib/gooddata/models/schedule.rb]: Is this right place for such function? What about having it in some helperl?

**Comment by korczis** [comment]
Merging, but see my comment.

**Pull request merged**